test/selenium/selenium-machines-network-page.py

```diff
@@ -0,0 +1,52 @@
+from testlib_avocado.seleniumlib import clickable, text_in, invisible
+from testlib_avocado.machineslib import MachinesLib
+
+
+class MachinesNetworkPageTestSuite(MachinesLib):
+    """
+    :avocado: enable
+    :avocado: tags=machines
+    """
+
+    def testNetworkPageInfo(self):
+        net_cmd_active = int(self.machine.execute('sudo virsh net-list | awk \'NR>=3{if($0!="")print}\' | wc -l')) + int(
+            self.machine.execute('virsh net-list | awk \'NR>=3{if($0!="")print}\' | wc -l'))
+        self.wait_css('#card-pf-networks > div > p > span:nth-child(1)',
+                      cond=text_in,
+                      text_=str(net_cmd_active))
+
+        total = int(self.wait_css(
+            '#card-pf-networks > div > p > span:nth-child(1)').text)
+        active = int(self.wait_css(
+            '#card-pf-networks > div > p > span:nth-child(1)').text)
```

Comment:
You have the same selector for total and active. One of them is wrong.

Reply:
Sorry, the selector of total is wrong, I will modify it